src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1003,27 +1002,31 @@ class SaagieClient {
 
     static getTemporaryFile(String url, boolean bool) {
         def tempJobDirectory = null
-
+    
+    
+        UUID uuid = UUID.randomUUID()
+        
         if(url){
```

Comment:
You don't seem to respect spaces along the file, don't forget to use CTRL + ALT + L to reformat code